File: esercizi_java/test/compito_python_4/Laboratorio.py
from Corso import Corso


class Laboratorio(Corso):
    def __init__(self, cod, nomecorso, nomedoc, nomelab, nomeass, numpos):
        super().__init__(cod, nomecorso, nomedoc)
        self._nomelab = nomelab
        self._nomeass = nomeass
        self._numpos = numpos

    def toString(self):
        return super().toString()+"lab.\t-\t-\t-\t"+self._nomelab+"\t"+self._nomeass+"\t"+str(self._numpos)

File: esercizi_java/NetBeansProjects/Universita/src/universita/Laboratorio.java
package universita;

/**
 *
 * @author tobiasacchetto
 */

public class Laboratorio extends Corso{
    private String nomelab,nomeass;
    private int numpos;
    
    public Laboratorio(int cod,String nomecorso,String nomedoc,String nomelab,String nomeass,int numpos){
        super(cod,nomecorso,nomedoc);
        this.nomelab=nomelab;
        this.nomeass=nomeass;
        this.numpos=numpos;
    }
    
    @Override
    public String toString(){
        return super.toString()+"lab.\t-\t-\t-\t"+nomelab+"\t"+nomeass+"\t"+numpos;
    }
}
